pkg/kates/client.go

```diff
@@ -462,6 +462,7 @@ func (c *Client) watchRaw(ctx context.Context, query Query, target chan rawUpdat
 			},
 		},
 	)
+	dlog.Errorf(ctx, "error adding event handlers, %s", err)
```

Comment:
shouldn't there be an if err != nil guard here?

Reply:
yes good catch. I will update.

Reply:
Done